src/Symfony/Component/Form/Extension/Validator/EventListener/ValidationListener.php

```diff
@@ -57,7 +57,6 @@ public function validateForm(FormEvent $event)
             foreach ($violations as $violation) {
                 // Allow the "invalid" constraint to be put onto
                 // non-synchronized forms
-                // ConstraintViolation::getConstraint() must not expect to provide a constraint as long as Symfony\Component\Validator\ExecutionContext exists (before 3.0)
                 $allowNonSynchronized = (null === $violation->getConstraint() || $violation->getConstraint() instanceof Form) && Form::NOT_SYNCHRONIZED_ERROR === $violation->getCode();
```

Comment:
well, not really though, as the component still supports Validator 2.8. This means the comment should be kept too